src/connectors/cacheDb/cacheDb.connector.js

```diff
@@ -68,9 +75,215 @@ const cacheRegistration = async registration => {
   }
 };
 
+/**
+ * Update the completed objects for the Registration and Tascomi objects
+ * @param {string} fsa_rn The FSA-RN for the registration to be updated
+ * @param {string} property The specific field to be updated
+ * @param {string} value The value for the result
+ */
+const updateCompletedInCache = async (fsa_rn, property, value) => {
+  logEmitter.emit(
+    "functionCall",
+    "cacheDb.connector",
+    "updateCompletedInCache"
+  );
+  try {
+    const cachedRegistrations = await establishConnectionToMongo();
+    let cachedRegistration = await cachedRegistrations.findOne({
+      "fsa-rn": fsa_rn
+    });
+
+    let newCompleted = cachedRegistration.completed;
```

Comment:
Likewise this should be a const, and since JS objects are passed by reference, it's a good idea to get in a habit of using Object.assign in these cases